contrib/debian/examples/bitcoin.conf

```diff
@@ -67,9 +67,30 @@
 # This option can be specified multiple times (default: bind to all interfaces)
 #rpcbind=<addr>
 
-# You must set rpcuser and rpcpassword to secure the JSON-RPC api
+# If no rpcpassword is set, rpc cookie auth is sought. The default `-rpccookiefile` name
```

Comment:
No, but more than one user was apparently confused by this!(getting feedback offline) Maybe it's too obvious.